charts/pact-broker/templates/deployment.yaml

```diff
@@ -93,10 +93,23 @@ spec:
             - name: PACT_BROKER_DATABASE_USERNAME
               value: {{ include "broker.databaseUser" . }}
             - name: PACT_BROKER_DATABASE_PASSWORD
+              {{- if or .Values.postgresql.auth.existingSecret .Values.externalDatabase.config.auth.existingSecret }}
```

Comment:
Is this needed if we have the `else` below?

Reply:
No, it's not necessary, removed!